app/schemas/analyze.py

```
@@ -0,0 +1,41 @@
+from fastapi import Body, Depends, Query, Path
```

Ruff (0.12.2)

`fastapi.Body` imported but unused

Remove unused import

(F401)